flowetl/etl/etl/etl_utils.py

```diff
@@ -114,6 +114,12 @@ def construct_etl_dag(
         schedule_interval=None,
         default_args=default_args,
         template_searchpath=config_path,  # template paths will be relative to this
+        user_defined_macros={
+            "get_extract_table": lambda execution_date: f"etl.x{ cdr_type }_{ execution_date }",
+            "get_transform_table": lambda execution_date: f"etl.t{ cdr_type }_{ execution_date }",
+            "get_load_table": lambda execution_date: f"events.{ cdr_type }_{ execution_date }",
+            "cdr_type": cdr_type.lower(),
```

Comment:
I see why you added the `.lower()` here but I think it should be fine to omit it and require the user to provide the correct capitalisation.
```suggestion
            "cdr_type": cdr_type,
```
(This will also avoid a discrepancy between the `cdr_type` macro and the value of `cdr_type` in the table macros above which don't use `.lower()`, although admittedly this is mostly a cosmetic detail).